Index.tsx

```diff
@@ -32,6 +35,40 @@ const Index: React.FC = () => {
                 <div className="demo-slide" style={{ width: '100px', height: '200px', backgroundColor: 'rgb(200, 200, 200)' }}>Slide 7</div>
                 <div className="demo-slide" style={{ width: '75px', height: '200px', backgroundColor: 'rgb(225, 225, 225)' }}>Slide 8</div>
             </Slider>
+            <h1>Slides with fixed width, initial on 3</h1>
+            <Slider hideNavigationButtons={hideNavigationButtons} initialSlideIndex={3}>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(50, 50, 50)' }}>Slide 1</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(75, 75, 75)' }}>Slide 2</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(100, 100, 100)' }}>Slide 3</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(125, 125, 125)' }}>Slide 4</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(150, 150, 150)' }}>Slide 5</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(175, 175, 175)' }}>Slide 6</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(200, 200, 200)' }}>Slide 7</div>
+                <div className="demo-slide demo-slide--fixed" style={{ backgroundColor: 'rgb(225, 225, 225)' }}>Slide 8</div>
+            </Slider>
+            <h1>Slides with fixed varying width, initial on 5</h1>
```

Comment:
Not sure if a second example with different configuration adds much value, a single demo showing the initial slide working would be good enough for me

Reply:
I used it to check if all works fine with variable width slides and when you push it to the end, so select a slide that doesn't have much space left on the end.